proposed/simplecache-meta.md

```diff
@@ -0,0 +1,82 @@
+PSR-Simple-Cache Meta Document
+==============================
+
+1. Summary
+----------
+
+Caching is a common way to improve the performance of any project, making
+caching libraries one of the most common features of many frameworks and
+libraries. Interoperability at this level means libraries can drop their
```

Comment:
Is the caching library a feature of many frameworks and libraries, or is utilising some form of caching a feature of most frameworks and libraries. This sentence isn't clear as to which you mean and I think you mean the latter.